### PR TITLE
Adapting atlasdb-cli to baseline checks

### PR DESCRIPTION
**Pull request by fsamuel-bs**
**Goals (and why)**:  Adapt the atlasdb-cli module to baseline requirements.

**Implementation Description (bullets)**: -

**Concerns (what feedback would you like?)**: If style is correctly followed on changes.

**Where should we start reviewing?**: -

**Priority (whenever / two weeks / yesterday)**: EOD

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/atlasdb/1852)

### CODE REVIEW

File: atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SingleBackendCommand.java [hunk 1]
```diff
@@ -39,17 +38,16 @@ public Integer call() {
         }
```
Comment (fsamuel-bs) [inline atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SingleBackendCommand.java]: We're printing the stack trace when catching the exception. This makes baseline not happy. Do we want to keep doing that?

File: atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/timestamp/FastForwardTimestamp.java [hunk 2]
```diff
@@ -25,8 +25,9 @@
 import io.airlift.airline.Command;
 
 @Command(name = "fast-forward", description = "Fast forward the stored upper limit of a persistent timestamp"
-        + " service to the specified timestamp.  Is used in the restore process to ensure that all future timestamps used are"
-        + " explicity greater than any that may have been used to write data to the KVS before backing up the underlying storage.")
+        + " service to the specified timestamp."
```
Comment (fsamuel-bs) [inline atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/timestamp/FastForwardTimestamp.java]: Not sure if this follows the style guidelines.

File: atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SingleBackendCommand.java (hunk 1 above)
Comment (fsamuel-bs) [inline atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SingleBackendCommand.java]: Discussed offline: behavior expected is to log stacktrace and print to System.err.

File: atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/timestamp/FastForwardTimestamp.java (hunk 2 above)
Comment (gsheasby) [inline atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/timestamp/FastForwardTimestamp.java]: I don't think we have consistent style guidelines. It looks ok to me.

File: atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SweepCommand.java [hunk 3]
```diff
@@ -70,11 +70,12 @@
...
 
     @Option(name = {"--cell-batch-size"},
-            description = "Sweeper cell batch size (default: " + AtlasDbConstants.DEFAULT_SWEEP_CELL_BATCH_SIZE+ ")")
+            description = "Sweeper cell batch size (default: " + AtlasDbConstants.DEFAULT_SWEEP_CELL_BATCH_SIZE + ")")
     int cellBatchSize = AtlasDbConstants.DEFAULT_SWEEP_CELL_BATCH_SIZE;
 
     @Option(name = {"--sleep"},
-            description = "Time to wait in milliseconds after each sweep batch (throttles long-running sweep jobs, default: 0)")
+            description = "Time to wait in milliseconds after each sweep batch "
+                    + "(throttles long-running sweep jobs, default: 0)")
```
Comment (jeremyk-91) [inline atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/SweepCommand.java]: nit: I think we normally have spaces at the beginning of strings that we concatenate i.e.

```
foo = "bar"
        + " baz";
```

File: atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/output/OutputPrinter.java [hunk 4]
```diff
@@ -18,6 +18,7 @@
 import org.slf4j.Logger;
 import org.slf4j.helpers.MessageFormatter;
 
+@SuppressWarnings("Slf4jConstantLogMessage")
```
Comment (jeremyk-91) [inline atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/output/OutputPrinter.java]: nit: Usually for these I prefer to explicitly state that these messages are from checkstyle, i.e. `@SuppressWarnings("checkstyle:Slf4jConstantLogMessage")`

File: atlasdb-cli/src/test/java/com/palantir/atlasdb/cli/runner/AbstractTestRunner.java [hunk 5]
```diff
@@ -59,8 +60,8 @@ public void freshCommand() throws URISyntaxException {
     }
 
     @Override
-    public void parse(String... args) {
-        this.args = Arrays.copyOf(args, args.length);
+    public void parse(String... arguments) {
+        this.args = Arrays.copyOf(arguments, arguments.length);
```
Comment (jeremyk-91) [inline atlasdb-cli/src/test/java/com/palantir/atlasdb/cli/runner/AbstractTestRunner.java]: Wow, didn't expect `args` to be a violation

Comment (gsheasby) [inline atlasdb-cli/src/test/java/com/palantir/atlasdb/cli/runner/AbstractTestRunner.java]: It hides the field `args`.

**Pull request merged**